tsconfig.json

```diff
@@ -48,7 +48,7 @@
       "@rainbow-me/performance/utils": ["src/performance/utils"],
       "@rainbow-me/analytics": ["src/analytics"],
       "ens-avatar": ["src/ens-avatar/src"],
-      "logger": ["src/utils/logger"],
+      "@/utils/logger": ["src/utils/logger"],
```

Comment:
Is this line now redundant since `@/utils/logger` already resolves to `src/utils/logger` via the generic `"@/*": ["./src/*"]` mapping?

Also, the PR description says you're leaving the config in place until all WIP code is merged, so should the original `logger` alias be left in for now?

Reply:
My find/replace was too strong 😄 great catch!